content/custom-commands/commands.md

```diff
@@ -239,6 +239,38 @@ You must specify a channel to run interval commands in even if the command doesn
 
 {{% /notice %}}
 
+#### Component
+
+{{% notice style="tip" %}}
+
+Use of interactions within YAGPDB is an advanced topic, the documentation should be used only as reference. To learn
+about using interactions, [see here](/reference/custom_interactions).
+
+{{% /notice %}}
+
+The component trigger is used to trigger custom commands via buttons or select menus.
+
+When creating a button or a select menu using template scripting, you must define a `custom_id`. When that button/menu
+is used by a member of your server, it triggers any custom command whose trigger matches that Custom ID.
+
+The trigger is matched using [RegEx](/reference/regex).
+
+#### Modal
+
+{{% notice style="tip" %}}
+
+Use of interactions within YAGPDB is an advanced topic, the documentation should be used only as reference. To learn
+about using interactions, [see here](/reference/custom_interactions).
+
+{{% /notice %}}
```

Comment:
This feels extremely repetitive to me — I understand that the current
layout doesn't give us much to work with; what do you think of
summarising these two headers under a collective "Interactions" header?

Reply:
Hmm, could we remove the warnings and turn them into hyperlinks "In-depth Interactions Guide"?

Reply:
Sure, that sounds like it could work.